src/inference/src/cpp_interfaces/interface/ie_iplugin_internal.cpp

```diff
@@ -529,4 +531,107 @@ void SetExeNetworkInfo(const std::shared_ptr<IExecutableNetworkInternal>& exeNet
     exeNetwork->setOutputs(const_results);
 }
 
+std::shared_ptr<::ov::IPlugin> convert_plugin(const std::shared_ptr<InferenceEngine::IInferencePlugin>& from) {
+    return ov::legacy_convert::convert_plugin(from);
+}
```

Comment:
should we move IPluginWrapper implementation to a dedicate file?

Reply:
I didn't move it because it is tiny implementation and will be removed at the same moment with IPluginInternal